ops/model.py

```diff
@@ -3756,7 +3770,7 @@ def add_metrics(
 
     def get_pebble(self, socket_path: str) -> pebble.Client:
         """Create a pebble.Client instance from given socket path."""
-        return pebble.Client(socket_path=socket_path)
+        return pebble.Client(socket_path=socket_path, app_id=self._juju_context.model_uuid)
```

Comment:
This is called "app_id", but it's the model UUID. Is that intended?

Reply:
No, I noticed that yesterday as well (in the spec). Will fix.